supportedBackends/jes/src/main/scala/cromwell/backend/impl/jes/JesJobCachingActorHelper.scala

```diff
@@ -37,20 +31,24 @@ trait JesJobCachingActorHelper extends StandardCachingActorHelper {
   lazy val jesStdoutFile: Path = jesCallPaths.stdout
   lazy val jesStderrFile: Path = jesCallPaths.stderr
   lazy val jesLogFilename: String = jesCallPaths.jesLogFilename
-  lazy val defaultMonitoringOutputPath: Path = callRootPath.resolve(JesMonitoringLogFile)
+
+  lazy val jesMonitoringParamName: String = JesJobPaths.JesMonitoringKey
+  lazy val localMonitoringLogPath: Path = DefaultPathBuilder.get(jesCallPaths.jesMonitoringLogFilename)
+  lazy val localMonitoringScriptPath: Path =  DefaultPathBuilder.get(jesCallPaths.jesMonitoringScriptFilename)
```

Comment:
Explained IRL. The local paths should be built using the `DPB`, and the docker paths built using the `JWD.MP`. Using the `DPB` versus something like `relativeLocalizationPath()` means that `/cromwell_root` will not be duplicated in the ultimate path. 👍